### ISSUE TITLE
Real server compatibility: x-ms-blob-sequence-number of 0 should be returned for HEAD requests on Page blob

### ISSUE THREAD

**Issue by ayeltsov** [Azure/Azurite#410]
### Which service(blob, file, queue, table) does this issue concern?
Blob
### Which version of the Azurite was used?
3.5
### Where do you get Azurite? (npm, DockerHub, NuGet, Visual Studio Code Extension)
GitHub sync to master
### What's the Node.js version?
n/a
### What problem was encountered?
The legacy emulator returns page blob sequence number of 0 for the HEAD request

HEAD http://127.0.0.1:10000/devstoreaccount1/sq2/ad_data.bak?st=<...>

Legacy:

![image](https://user-images.githubusercontent.com/9418663/75379256-f278fa80-5889-11ea-82a1-6eac5590e97f.png)

Azurite:

![image](https://user-images.githubusercontent.com/9418663/75379277-fb69cc00-5889-11ea-9ee3-c5a152954575.png)

### Steps to reproduce the issue?
Sniff network for legacy vs. Azurite (or for Azurite, review debug log)

### Have you found a mitigation/solution?

Not sure yet. We have native code library that works directly with REST API and it asserts that x-ms-blob-sequence-number is sent for page blob.

Documentation does not sound like this header is optional (and even if it did, we should probably maintain compatibility):
https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob-properties

<...>
x-ms-blob-sequence-number 
The current sequence number for a page blob.
This header is not returned for block blobs or append blobs.
This header is not returned for block blobs.
<...>

I would appreciate suggestions for temporary hacks in the Emulator code.

**Comment by ayeltsov** [comment]
Found a hack (or maybe even fix, please evaluate) that works for my scenario.

https://github.com/Azure/Azurite/blob/be4dc2c9d44e3ba643e616795c26cb48dedd5c8e/src/blob/handlers/PageBlobHandler.ts#L117

```
        blobSequenceNumber: options.blobSequenceNumber,
=>
        blobSequenceNumber: options.blobSequenceNumber ? options.blobSequenceNumber : 0,

```

This results in future HEAD and [at least some but engouh satisfy our caller] GET/PUT requests having x-ms-blob-sequence-number with value 0

**Comment by blueww** [comment]
@ayeltsov
Thanks for reporting this issue!
We will prioritize the fix of it in our future release.

**Comment by XiaoningLiu** [comment]
Fixed in v3.6.0. Will close this issue.
